projects/packages/search/src/dashboard/components/pages/dashboard-page.jsx

```diff
@@ -144,7 +144,44 @@ export default function DashboardPage( { isLoading = false } ) {
 	);
 }
 
-const MockUsageMeter = ( { sendPaidPlanToCart } ) => {
+const PlanSummary = ( { latestMonthRequests } ) => {
+	const tierSlug = useSelect( select => select( STORE_ID ).getTierSlug() );
+
+	const startDate = new Date( latestMonthRequests.start_date );
+	const endDate = new Date( latestMonthRequests.end_date );
+
+	const localeOptions = {
+		month: 'short',
+		day: '2-digit',
+	};
+
+	// Leave the locale as `undefined` to apply the browser host locale.
+	const startDateText = startDate.toLocaleDateString( undefined, localeOptions );
+	const endDateText = endDate.toLocaleDateString( undefined, localeOptions );
+
+	let planText = __( 'Paid Plan', 'jetpack-search-pkg' );
+	if ( ! tierSlug ) {
+		planText = __( 'Free Plan', 'jetpack-search-pkg' );
+	}
+
+	return (
+		<h2>
+			{ createInterpolateElement(
+				sprintf(
+					// translators: %1$s: Usage period, %2$s: Plan name
+					__( 'Your usage <s>%1$s (%2$s)</s>', 'jetpack-search-pkg' ),
+					`${ startDateText }-${ endDateText }`,
```

Comment:
```suggestion
					`${ startDateText } - ${ endDateText }`,
```

I know this suggestion deviates from design, so I'll leave this implementation to your discretion.

Reply:
I consider leaving spaces around `-` is more graceful as well. 👍🏼